DEVELOP.md

```diff
@@ -1,48 +1,101 @@
 # Primer Development
 
-If you've made it this far, **thank you**! We appreciate your contribution, and hope that this document helps you along the way.
+If you've made it this far, **thank you**! We appreciate your contribution, and hope that this document helps you along the way. If you have any questions or problems, don't hesitate to [reach out to us on Spectrum](https://spectrum.chat/primer).
```

Comment:
We shouldn't point people to reach out to us on Spectrum since we rarely check it between office hours. Let's point them to leaving an issue in the repo instead.